e2e/content-services/permissions-component.e2e.ts

```diff
@@ -245,4 +261,56 @@ describe('Permissions Component', function () {
         });
 
     });
+
+    describe('Role Consumer', function () {
+
+        beforeEach(async (done) => {
+
+            loginPage.loginToContentServicesUsingUserModel(fileOwnerUser);
+            contentServicesPage.goToDocumentList();
+
+            contentList.checkContentIsDisplayed(roleFolder.name);
+            contentServicesPage.checkSelectedSiteIsDisplayed('My files');
+            contentList.clickRowMenuActionsButton(roleFolder.name);
+            contentList.clickMenuActionNamed('PERMISSION');
+            permissionsPage.checkAddPermissionButtonIsDisplayed();
+            permissionsPage.clickAddPermissionButton();
+            permissionsPage.checkAddPermissionDialogIsDisplayed();
+            permissionsPage.checkSearchUserInputIsDisplayed();
+            permissionsPage.searchUserOrGroup(filePermissionUser.getId());
+            permissionsPage.clickUserOrGroup(filePermissionUser.getFirstName());
+            permissionsPage.checkUserOrGroupIsAdded(filePermissionUser.getId());
+            permissionsPage.clickRoleDropdown();
+            permissionsPage.selectOption('Consumer');
+            done();
+        });
+
+        it('[C274691] Should not to be able to Delete, see Info or Upload a file with Consumer permission', () => {
+
+            loginPage.loginToContentServicesUsingUserModel(filePermissionUser);
+            contentServicesPage.goToDocumentList();
+            browser.sleep(15000); // for the Search indexing to finish
+            searchDialog
+                .checkSearchIconIsVisible()
+                .clickOnSearchIcon()
+                .checkSearchBarIsVisible()
+                .enterText(roleFolderModel.name)
+                .resultTableContainsRow(roleFolderModel.name)
+                .clickOnSpecificRow(roleFolderModel.name);
+            contentList.checkContentIsDisplayed('Role' + fileModel.name);
+            contentList.doubleClickRow('Role' + fileModel.name);
+            viewerPage.checkFileIsLoaded();
+            viewerPage.clickCloseButton();
+            contentList.waitForTableBody();
+            contentList.checkDeleteIsDisabled('Role' + fileModel.name);
+            browser.refresh();
```

Comment:
Is this refresh() needed?

Reply:
I have removed browser refresh and added the Protractor Escape key and a check to make sure, the action menu is not displayed anymore, before carrying on furtheractions.